### DISCUSSION TITLE
Ubuntu 18.04.6 LTS (GNU/Linux 4.15.0-210-generic x86_64

### DISCUSSION THREAD

**Discussion by HalowTV** [streamlink/streamlink#5479] [Q&A] [answered]
### Description

pls help me how to updatet to last version?
Hi  i run  streamlink -V  i get version streamlink 0.10.0

and i try updatet whit  
sudo pip3 install --upgrade streamlink
error
sudo: pip3: command not found
and i run all this commando 
pip install --user -U streamlink
and is get error to 
pip install --user -U streamlink
-bash: pip: command not found

**Comment by bastimeyer** [comment]
- https://streamlink.github.io/install.html#linux-and-bsd
- https://streamlink.github.io/install.html#package-availability

**Reply by HalowTV** [reply]
thx but i get still this version 
root@Ubuntu-1804-bionic-64-minimal ~ # sudo apt install streamlink
Reading package lists... Done
Building dependency tree
Reading state information... Done
streamlink is already the newest version (0.10.0+dfsg-1).
0 upgraded, 0 newly installed, 0 to remove and 10 not upgraded.
root@Ubuntu-1804-bionic-64-minimal ~ # streamlink -V
streamlink 0.10.0

**Reply by bastimeyer** [reply]
You clearly didn't read the docs I've just linked.

**Reply by HalowTV** [reply]
i well do mate but i get only error

**Reply by HalowTV** [reply]
when i run this get version 3
sudo apt install python3-pip

python3 -m pip install streamlink==3.0.3

but try updatet to version 6 get error

**Reply by bastimeyer** [reply, marked as answer]
If you had actually read the docs, then you would have seen this

> **Packaging is not done by the Streamlink maintainers themselves** except for the [PyPI package](https://streamlink.github.io/install.html#id4), the [Windows installers + portable builds](https://streamlink.github.io/install.html#windows-binaries), and the [Linux AppImages](https://streamlink.github.io/install.html#linux-appimages).

1. Debian != Ubuntu
2. Streamlink's docs do not advertise Ubuntu support
3. Debian's package version is currently [`5.2.1 on sid/unstable`](https://packages.debian.org/unstable/streamlink) and [`5.5.1 on experimental`](https://packages.debian.org/experimental/streamlink)
4. Whatever [Ubuntu](https://packages.ubuntu.com/mantic/python3-streamlink) or other downstream distros of Debian do is up to those distros

As you can clearly see, Debian is already lagging behind because of their release schedule, and any downstream distro of Debian will lag even more behind, which is why we don't advertise any support for those distros.

> If a packaged release of Streamlink is not available for your operating system / distro or your system's architecture, or if it's out of date or broken, then please contact the respective package maintainers or package-repository maintainers of your operating system / distro, as it's up to them to add, update, or fix those packages.
> **Users of glibc-based Linux distros can find up-to-date Streamlink releases via the available [AppImages](https://streamlink.github.io/install.html#linux-appimages).**

Good luck.
